tests/platform_tests/sfp/test_sfputil.py

```diff
@@ -30,6 +39,144 @@
 ]
 
 
+class LogicalInterfaceDisabler:
+    """
+    Disable the given logical interface and restore afterwards.
+    """
+    def __init__(self, duthost, enum_frontend_asic_index, logical_intf, phy_intf,
+                 skip_dom_polling_handle=False, skip_lpmode_handle=False):
+        self.duthost = duthost
+        self.logical_intf = logical_intf
+        self.phy_intf = phy_intf
+        self.original_lpmode_state = None
+        self.skip_dom_polling_handle = skip_dom_polling_handle
+        self.skip_lpmode_handle = skip_lpmode_handle
```

Comment:
@longuan-cisco We need to skip lpmode config for all CMIS cables as well as cables which are `Power Class 1 Module`. Can you please handle this scenario?

Reply:
Added the handling for both